enumerable.rb

```diff
@@ -0,0 +1,52 @@
+module MyEnumerable
+  def all?(&block)
+    each do |element|
+      return false unless block.call(element)
+    end
+    true
+  end
+
+  def any?(&block)
+    each do |element|
+      return true if block.call(element)
+    end
+    false
+  end
+
+  def filter(&block)
+    result = []
+    each do |element|
+      result << element if block.call(element)
+    end
+    result
+  end
+end
+
+class MyList
+  include MyEnumerable
+  def initialize(*elements)
+    @list = elements
+  end
+
+  def each(&block)
+    @list.each do |element|
+      block.call(element)
+    end
+  end
+end
```

Comment:
Nice job so far. However, as part of the project requirements, consider putting the class `MyList` and the module `MyEnumerable` in a separate .rb file. This will help make your code more cleaner and maintainable. :pray: